stock_indicators/indicators/parabolic_sar.py

```diff
@@ -0,0 +1,74 @@
+from typing import Iterable, Optional, Type, TypeVar, overload
+from stock_indicators._cslib import CsIndicator
+from stock_indicators._cstypes import List as CsList
+from stock_indicators._cstypes import Decimal as CsDecimal
+from stock_indicators._cstypes import to_pydecimal
+from stock_indicators.indicators.common.results import IndicatorResults, ResultBase
+from stock_indicators.indicators.common.quote import Quote
+
+@overload
+def get_parabolic_sar(quotes: Iterable[Quote],
+                      acceleration_step: float = 0.02,
+                      max_acceleration_factor: float = 0.2
+                      ) -> "ParabolicSARResults[ParabolicSARResult]": ...
+# TODO: Uncomment when #82 is fixed.
+# @overload
+# def get_parabolic_sar(quotes: Iterable[Quote],
+#                       acceleration_step: float,
+#                       max_acceleration_factor: float,
+#                       initial_factor: float
+#                       ) -> "ParabolicSARResults[ParabolicSARResult]": ...
+def get_parabolic_sar(quotes,
+                      acceleration_step = 0.02,
+                      max_acceleration_factor = 0.2,
+                      initial_factor = None):
+    initial_factor = initial_factor if initial_factor else acceleration_step
```

Comment:
unused variable

Reply:
It was for the future release. But I'll remove for now.